src/components/PostComment.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { useParams } from "react-router"
+import { postComment } from "../api";
+
+export const CommentPoster = ({comments, setComments}) => {
+    const [name, setName] = useState([])
+    const [body, setBody] = useState([])
+    const {review_id} = useParams()
+    
+    const handlePost = (event) => {
+        event.preventDefault();
+        const newComment = {
+            author: name,
+            body:body,
```

Comment:
I think a hard coded user would be better here for now rather than making me put a username